org.eclipse.buildship.core.test/src/main/groovy/org/eclipse/buildship/core/workspace/internal/ImportingWtpProjects.groovy

```diff
@@ -23,6 +24,34 @@ class ImportingWtpProjects extends ProjectSynchronizationSpecification {
     private static final String NON_DEPLOYED = "org.eclipse.jst.component.nondependency"
     private static final String DEPLOYED = "org.eclipse.jst.component.dependency"
     private static final String WTP_COMPONENT_NATURE = "org.eclipse.wst.common.modulecore.ModuleCoreNature"
+	
+	def "Do not check mixed deployment paths without WTP installed"() {
+		setup:
+		File root = dir("project") {
+			file 'build.gradle', """
+			    apply plugin: 'java'
+                apply plugin: 'ear'
+                ${jcenterRepositoryBlock}
+                dependencies {
+                    earlib 'org.assertj:assertj-core:3.10.0'
+                    deploy "log4j:log4j:1.2.17"
+                }
+            """
+		}
+		
+		UserNotification notification = Mock(UserNotification)
+		registerService(UserNotification, notification)
+
+		WorkspaceOperations operations = new WorkspaceOperationsDelegate(recognizeWtpComponentNature: false)
+		registerService(WorkspaceOperations, operations)
+
+		when:
+		importAndWait(root)
+
+		then:
+        0 * notification.errorOccurred(*_)
```

Comment:
Wrong indentation

Reply:
@moley for this project, set up your formatter to be the same as this repository. If you don't use one, I can recommend it.